users/donoskaro.json

```diff
@@ -0,0 +1,8 @@
+{
+    "copyright": "Oskar Augustyn“,
```

Comment:
Heya @donoskaro , looks like the `“` character isn't a legit `"` double quote, and is breaking the build. You may want to switch `“` with `"` here 🤙

Reply:
Oh didn't notice that @squanto, apologies for this.